app/src/main/java/org/gdg/frisbee/android/app/GoogleApiClientFactory.java

```diff
@@ -0,0 +1,37 @@
+package org.gdg.frisbee.android.app;
+
+import android.content.Context;
+
+import com.google.android.gms.appindexing.AppIndex;
+import com.google.android.gms.common.api.GoogleApiClient;
+import com.google.android.gms.drive.Drive;
+import com.google.android.gms.games.Games;
+import com.google.android.gms.plus.Plus;
+
+import org.gdg.frisbee.android.utils.PrefUtils;
+
+public final class GoogleApiClientFactory {
+    private GoogleApiClientFactory() { }
```

Comment:
@friedger Do you mean an issue to discuss/create a convention for the order of methods or for this specific case?